ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/snmp_settings.schema.yml

```
@@ -0,0 +1,71 @@
+# yaml-language-server: $schema=../../../../plugins/plugin_utils/schema/avd_meta_schema.json
+# Line above is used by RedHat's YAML Schema vscode extension
+# Use Ctrl + Space to get suggestions for every field. Autocomplete will pop up after typing 2 letters.
+type: dict
+keys:
+  snmp_settings:
+    type: dict
+    description: Set SNMP settings (optional).
+    keys:
+      contact:
+        type: str
+        description: SNMP contact.
+      location:
+        type: bool
+        default: False
+        description: Set SNMP location. Formatted as {{ fabric_name }} {{ dc_name }} {{ pod_name }} {{ switch_rack }} {{ inventory_hostname }}.
+      compute_local_engineid:
+        type: bool
+        default: False
+        description: |
+          Generate a local engineId for SNMP by hashing via SHA1, the string
+          generated via the concatenation of the hostname plus the management IP.
+          {{ inventory_hostname }} + {{ switch.mgmt_ip }}
```

Fixing to latest description (from current doc) - not sure why it was changed?
```suggestion
          Generate a local engineId for SNMP using the 'compute_local_engineid_source' method.
```